src/main/java/seedu/address/model/person/Module.java

```diff
@@ -4,15 +4,15 @@
 import static seedu.address.commons.util.AppUtil.checkArgument;
 
 /**
- * Represents a Person's address in the address book.
- * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
+ * Represents a Person's module in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidModule(String)}
  */
-public class Address {
+public class Module {
 
     public static final String MESSAGE_CONSTRAINTS = "Addresses can take any values, and it should not be blank";
```

Comment:
I think you should change the message constraint to be about the module instead, maybe something like should be 2 alphabets with 4 numbers "qqxxxx"

Reply:
Thanks for pointing it out, will resolve it!